lametro/people.py

```diff
@@ -98,6 +100,9 @@ def scrape(self):
             person_api_url, person_web_url = source_urls
             p.add_source(person_api_url, note='api')
             p.add_source(person_web_url, note='web')
+            p.family_name = term['OfficeRecordLastName']
+            p.given_name = term['OfficeRecordFirstName']
+
```

Comment:
@fgregg What's going on here? This seems to be duplicating (or overriding) attributes set in L68-69?

Reply:
you are right. this can be removed.